app/public/css/styles.css

```diff
@@ -889,6 +891,45 @@ button[data-balloon] {
   text-align: center;
 }
 
+/* BANNER */
+
+.banner {
+  max-width: var(--width);
+  width: 100%;
+  margin: 0 auto;
+  padding: 0 16px;
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: -100px;
+  z-index: 10;
+}
+
+.banner-inner {
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px 24px;
+}
+
+.banner-text::before {
+  content: '';
+  width: 24px;
+  height: 24px;
+  display: inline-block;
+  margin-right: 16px;
+  vertical-align: middle;
+  background: url('../assets/icon-star.svg') no-repeat;
+  background-position: center center;
+}
+
+.banner-close {
+  float: right;
```

Comment:
The `float` is not needed, if you want to make extra sure it stays stuck to the right side you can add `flex: 1` to `.banner-text`

Reply:
Great idea, thanks! Fixed